docs/general/endpoints.rst

```diff
@@ -0,0 +1,163 @@
+              {'asset': 'ASSET-2', 'name': 'TAG-2'}],
+     'time-seconds': '0.0165'}
+
+/anomaly/prediction
+===================
+
+The ``/anomaly/prediction`` endpoint will return the data supplied by the ``/prediction`` endpoint
+but reserved for models which output the same shape as their input, expected to be AutoEncoder type
+models.
+
+By this restriction, additional _features_ are calculated and returned:
+
+- ``error-transformed``:
+    - Error per feature/tag calculated from the transformed values
+- ``error-untransformed``:
+    - Error per feature/tag calculated from the ``inverse-transformed-model-output`` against ``original-input``
+- ``inverse-transformed-model-output``:
+    - The ``model-output`` which is ran backward through pipeline, which "untransforms" it to the same level
+      as the ``original-input``; this assumes that all steps in the pipeline support ``.inverse_transform``
+- ``total-transformed-error``:
+    - This is the vector norm of the error between a sample output and the sample input, using the transformed values
+- ``total-untransformed-error``:
+    - This is the vector norm of the error between a sample output and the sample input, using the inverse-transformed values
+
+Sample response:
+
+.. code-block:: python
+
+    {'data': [{'end': [None],
+               'error-transformed': [2.746687859183499,
+                                     2.4497416272485886,
+                                     2.508896707372706],
+               'error-untransformed': [36.94235610961914,
+                                       33.25593566894531,
+                                       34.43174743652344],
+               'inverse-transformed-model-output': [37.94235610961914,
+                                                    35.25593566894531,
+                                                    37.43174743652344],
+               'model-output': [0.6726480722427368,
+                                0.5305765271186829,
+                                0.7146536707878113],
+               'original-input': [1, 2, 3],
+               'start': [None],
+               'total-transformed-error': [4.454221696809242],
+               'total-untransformed-error': [60.4668517758241],
+               'transformed-model-input': [-2.0740397869407623,
+                                           -1.919165100129906,
+                                           -1.7942430365848951]}],
+     'tags': [{'asset': 'ASSET-A', 'name': 'TAG-1'},
+              {'asset': 'ASSET-B', 'name': 'TAG-2'},
+              {'asset': 'ASSET-C', 'name': 'TAG-3'}],
+     'time-seconds': '0.0866'}
+
+
+/download-model
+===============
+
+Returns the current model being served. Loadable via ``gordo_components.serializer.loads(downloaded_bytes)``
+
+
+/metadata
+=========
+
+Various metadata surrounding the current model and environment.
+
+
+/
+=
+
+This is the Swagger UI for the given model
+
+
+Project index page
```

Comment:
Maybe a comment noting that this is the output from watchman, which by default is routed at ... ?